index.js

```diff
@@ -35,6 +35,14 @@ module.exports.pitch = function (request) {
 		"}"
 	].join("\n");
 
+        var insertInto = "void 0";
+        if (typeof options.insertInto === "function") {
+          insertInto = options.insertInto.toString();
+        }
+        if (typeof options.insertInto === "string") {
```

Comment:
This check shouldn't be necessary as there isn't anything to handle within the pitching phase of the loader in case `options.insertInto` is a `{String}`

Reply:
I just want to make this check explicit, if you think that this is excess, I can delete this)